packages/manager/modules/billing/src/orders/billing-orders-main.html

```diff
@@ -0,0 +1,32 @@
+<oui-header heading="{{:: 'orders_page_title' | translate }}">
+    <oui-guide-menu
+        data-text="{{:: 'orders_guides' | translate }}"
+        data-ng-if="$ctrl.billingGuideUrl"
+    >
+        <oui-guide-menu-item
+            data-ng-href="{{:: $ctrl.billingGuideUrl }}"
+            data-external="true"
+        >
+            <span data-translate="orders_billing"></span>
+        </oui-guide-menu-item>
+        <oui-guide-menu-item
+            data-ng-if="$ctrl.featuresAvailabilities['billing:purchases-order']"
+            data-ng-href="{{:: $ctrl.purchaseOrdersGuideUrl }}"
+            data-external="true"
+        >
+            <span data-translate="orders_purchase"></span>
+        </oui-guide-menu-item>
+    </oui-guide-menu>
+    <oui-header-tabs
+        data-ng-if="$ctrl.featuresAvailabilities['billing:purchases-order']"
+    >
+        <oui-header-tabs-item state="app.account.billing.orders.orders"
+            ><span data-translate="orders_page_title"></span
+        ></oui-header-tabs-item>
+        <oui-header-tabs-item state="app.account.billing.orders.purchases"
```

Comment:
same as her: https://github.com/ovh/manager/pull/6441/files#r809775017

Reply:
My point is to have something like this:
```suggestion
        <oui-header-tabs-item state="{{:: $ctrl.orderPurchasesState}}"
```